client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/contact-form/index.tsx

```diff
@@ -103,7 +121,11 @@ const SignupContactForm = ( { onContinue }: Props ) => {
 				/>
 			</FormField>
 
-			<FormField label={ translate( 'Business URL' ) } isRequired>
+			<FormField
+				error={ validationError.agencyUrl }
+				label={ translate( 'Business URL' ) }
```

Comment:
@andrii-lysenko is there a reason why we are not using the checks property provided from the `withErrorHandling` HOC?

You can see example usage [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/a8c-for-agencies/sections/partner-directory/agency-details/index.tsx#L236):

```
<FormField
					label={ translate( 'Company location' ) }
					error={ validationError.country }
					checks={ [ validateNonEmpty() ] }
					field={ formData.country }
					isRequired
				>
					<SearchableDropdown
						value={ formData.country }
						onChange={ ( value ) => {
							setFormFields( { country: value } );
							updateValidationError( { country: undefined } );
						} }
						options={ countryOptions }
						disabled={ false }
					/>
				</FormField>
```